[prev](protein.md) | [toc](./README.md) | [next](ncbi.md)

# Which BLAST to use

Blast can also be used to align two sequences, but is most commonly used to compare a
sequence against all entries in a database of sequences.

The NCBI BLAST website has a few variants that people can use:

* [Nucleotide BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) (blastn): search for similar nucleotide sequences in a nucleotide sequence database
* [blastx](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastx&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome): search protein databases using a translated nucleotide query
* [tblastn](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=tblastn&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome): search translated nucleotide databases using a protein query
* [Protein BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&PAGE_TYPE=BlastSearch&LINK_LOC=blasthome) (blastp): search for similar protein sequences in a protein sequence database

Or, how NCBI visualizes it themselves:

![](blast_types.png)

[prev](protein.md) | [toc](./README.md) | [next](ncbi.md)
